api/question/models.py

```diff
@@ -13,3 +13,4 @@ class Question(Base, Utility):
     end_date = Column(String, nullable=False)
     total_views = Column(Integer, default=0)
     response = relationship('Response', cascade="all, delete-orphan")
+    is_active = Column(Boolean, nullable=True)
```

Comment:
I think for this it would be better if you set a default value to false rather than setting nullable to true